### PR TITLE
Tc 4015

### PR DESCRIPTION
**Pull request by sasha-3C**
Disabling start button after clicking to prevent duplicate form submissions

### CODE REVIEW

File: js/apps/thirdchannel/views/checkins/task_list.js [hunk 1]
```diff
@@ -31,6 +32,11 @@ define(function(require){
                 this.showError(e);
             }
         },
+        disableStart: function(e) {
+            e.preventDefault();
+            $(e.currentTarget)[0].disabled = true;
+            this.$('form.checkin-action').submit();
```
Comment (spullen) [inline js/apps/thirdchannel/views/checkins/task_list.js]: Is `'form.checkin-action'` the same thing as `'.checkin-form-btn'`?

Comment (sasha-3C) [inline js/apps/thirdchannel/views/checkins/task_list.js]: don't think so - I think 'form.checkin-action' is the whole form, other one is just the submit button

Comment (spullen) [inline js/apps/thirdchannel/views/checkins/task_list.js]: Yeah, ended up catching it, see updated comment.

Comment (sasha-3C) [inline js/apps/thirdchannel/views/checkins/task_list.js]: Tested that code and it doesn't seem to work... only disables the button and then doesn't do anything afterwards.

Comment (spullen) [inline js/apps/thirdchannel/views/checkins/task_list.js]: Fair enough

**Comment by spullen** [comment]
Looks fine to me.

**Pull request merged**